app/views/email_reply_mailer/mention.text.erb

```diff
@@ -3,3 +3,6 @@
   <%= word_wrap(@comment.plaintext_comment, :line_width => 72).gsub(/\n/, "\n  ") %>
 
 Continue this discussion at <%= @comment.url %>
+<p>
+Also, you can reply to this email to join  the discussion.
+</p>
```

Comment:
```suggestion
Reply to this email or visit <%= @comment.url %>
```

These mailers aren't HTML and we shouldn't invite the user to join the discussion because they might already be active in it when they're mentioned.